theme/reviews/reviewTiles.css

```diff
@@ -151,27 +153,29 @@
 }
 
 .review__narrative_course-report {
-  @apply col-span-2 grid grid-rows-3 grid-cols-1 p-5;
-  @apply md:p-5;
+  @apply col-span-2 grid grid-cols-1 grid-rows-1 place-items-start p-5;
+  @apply md:p-5 md:max-h-36;
 
   & .narrative__text {
-    @apply col-start-1 col-span-1 row-start-1 row-span-2;
+   @apply h-full w-full z-0 row-start-1 col-start-1;
+   @apply md:pb-5 md:overflow-hidden;
   }
 
   & .narrative__read-more {
-    @apply bg-gradient-to-t col-start-1 col-span-1 row-start-1 row-span-2 w-full h-full;
+    @apply bg-gradient-to-t w-full h-full z-10 row-start-1 col-start-1;
```

Comment:
can we eliminate `row-start-1` and `col-start-1` here? aren't those implicit if we're following a normal flow of a grid?